app/views/challenges/drawing.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= render partial: 'challenges/header', cache: true %>
```

Comment:
I assume this is just a bit of editorializing? E.g. we don't have this on the example page: https://redesignchallenge.org/drawing